genai-perf/genai_perf/parser.py

```diff
@@ -168,16 +166,16 @@ def _check_conditional_args(
     """
 
     # Endpoint and output format checks
-    if args.service_kind == "openai":
+    if args.service_kind in ["openai"]:
```

Comment:
I think you forgot to undo this change :)

Reply:
Good catch, thanks!